pathway-analysis/qusage_single_dataset.Rmd

```diff
@@ -312,7 +337,8 @@ We can look at the distribution of genes in this pathway with the
 We tell this function which pathway to plot with the `path.index` argument.
 
 ```{r}
-grep("ECM_RECEPTOR", names(kegg_pathways))
+grep("ECM_RECEPTOR", # Replace with desired pathway -- names of pathways can be seen using the `names(kegg_pathways)` function
```

Comment:
Bring wording from here to this section. We should add a little more explanation about what this is doing and how `grep` is looking for a pattern.